src/ir_LG.cpp

```diff
@@ -47,6 +47,16 @@ const uint16_t kLg2HdrSpace = 9900;           ///< uSeconds.
 const uint16_t kLg2BitMark = 480;             ///< uSeconds.
 
 const uint32_t kLgAcAKB74955603DetectionMask = 0x0000080;
+const uint8_t  kLgAcChecksumSize = 4;  ///< Size in bits.
```

Comment:
From below ?
// The 16 bits before the checksum.

Reply:
That's not really related. It is just something to make it look "nice" in `address` & `command` components of `result`.
The 16 bits really is arbitrary (but nice) in that respect.